src/main/java/view/Menu.java

```diff
@@ -1,4 +1,22 @@
 package view;
 
+import view.impl.UsersMenu;
+
+import java.util.Scanner;
+
 public class Menu {
+    public void runUsersMenu() {
+        System.out.println("Select main menu item:");
+        System.out.println("1. Login Menu");
+        System.out.println("2. User Main Menu");
+        System.out.println("3. User Product Menu");
+        System.out.println("4. Users Menu");
+
+        Scanner scanner = new Scanner(System.in);
+        int input = scanner.nextInt();
+        if (input == 4) {
```

Comment:
4 is magic number

Reply:
fixed. removed magic numbers and created constants